File: app/src/main/java/com/listenergao/audioandvideolearning/utils/ToastUtils.java
package com.listenergao.audioandvideolearning.utils;

import android.widget.Toast;

import com.listenergao.audioandvideolearning.LearningApplication;

public class ToastUtils {


    public static void toast(int resId) {
        toast(LearningApplication.getContext().getString(resId), Toast.LENGTH_SHORT);
    }

    public static void toast(String msg) {
        toast(msg, Toast.LENGTH_SHORT);
    }

    public static void toast(int resId, int duration) {
        toast(LearningApplication.getContext().getString(resId), duration);
    }

    private static void toast(String msg, int duration) {
        Toast.makeText(LearningApplication.getContext(), msg, duration).show();
    }
}

File: app/src/main/java/com/listenergao/audioandvideolearning/utils/ToastUtils.kt
package com.listenergao.audioandvideolearning.utils

import com.listenergao.audioandvideolearning.utils.ToastUtils
import com.listenergao.audioandvideolearning.LearningApplication
import android.widget.Toast

/**
 * @author listenergao
 */
object ToastUtils {

    fun toast(resId: Int) {
        toast(LearningApplication.getContext().getString(resId), Toast.LENGTH_SHORT)
    }

    fun toast(msg: String) {
        toast(msg, Toast.LENGTH_SHORT)
    }

    fun toast(resId: Int, duration: Int) {
        toast(LearningApplication.getContext().getString(resId), duration)
    }

    fun toast(msg: String, duration: Int) {
        Toast.makeText(LearningApplication.getContext(), msg, duration).show()
    }
}